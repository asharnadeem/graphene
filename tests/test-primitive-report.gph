int main()
{
    int a = 10;
    int b = 5;

    float c, d;
    c = 0.5;
    d = 0.25;

    string e = "foo";
    prints(e);

    print(a + b);
    printf(c + d); 
    
    print(a - b); 
    printf(c - d); 
    
    print(a * b); 
    printf(c * d); 
    
    print(a / b); 
    printf(c / d);
}
