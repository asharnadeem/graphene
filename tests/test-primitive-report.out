foo
15
0.75
5
0.25
50
0.125
2
2